openedx_tagging/core/tagging/models/utils.py

```diff
@@ -1,15 +1,16 @@
 """
 Utilities for tagging and taxonomy models
 """
-from django.db.models import Aggregate, CharField
-from django.db.models.expressions import Func
+from django.db import connection as db_connection
+from django.db.models import Aggregate, CharField, TextField
+from django.db.models.expressions import Combinable, Func
 
 RESERVED_TAG_CHARS = [
     '\t',  # Used in the database to separate tag levels in the "lineage" field
            # e.g. lineage="Earth\tNorth America\tMexico\tMexico City"
     ' > ',  # Used in the search index and Instantsearch frontend to separate tag levels
             # e.g. tags_level3="Earth > North America > Mexico > Mexico City"
-    ';',   # Used in CSV exports to separate multiple tags from the same taxonomy
+    ';',   # Used in CSV exports to separate multiple tags from the same taxonomy…
```

Comment:
```suggestion
    ';',   # Used in CSV exports to separate multiple tags from the same taxonomy
```
Not sure why this ellipsis character was added here.